app/components/ui/my-domains/new-domain-card/index.js

```diff
@@ -1,23 +1,22 @@
 // External dependencies
 import i18n from 'i18n-calypso';
-import { Link } from 'react-router';
 import React from 'react';
 import withStyles from 'isomorphic-style-loader/lib/withStyles';
 
 // Internal dependencies
-import { getPath } from 'routes';
+import config from 'config';
 import styles from './styles.scss';
 
 const NewDomainCard = () => {
+	const newSearchUrl = config( 'new_search_url' );
 	return (
-		<Link className={ styles.newDomainCard } to={ getPath( 'search' ) }>
+		<a className={ styles.newDomainCard } href={ newSearchUrl }>
 			<div>
-				<svg className={ styles.gridicon } height="48" width="48" viewBox="0 0 24 24">
-					<g><path d="M12 2C6.477 2 2 6.477 2 12s4.477 10 10 10 10-4.477 10-10S17.523 2 12 2zm5 11h-4v4h-2v-4H7v-2h4V7h2v4h4v2z"></path></g>
-				</svg>
-				<span>{ i18n.translate( 'Find a New Domain' ) }</span>
+				<span>
+					{ i18n.translate( 'Use wordpress.com/domains to find a new domain.' ) }
```

Comment:
I'd break so `wordpress.com/domains` is on a separate line, or kill the ALL CAPS as it looks bad here.